.travis.yml

```diff
@@ -9,33 +11,40 @@ branches:
   only:
     - master
 
-sudo: false
-
-php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-
-env:
-  - WP_VERSION=latest WP_MULTISITE=0
-  - WP_VERSION=latest WP_MULTISITE=1
+cache:
+  - composer
+  - $HOME/.composer/cache
 
 matrix:
   include:
+    - php: 7.1
+      env: WP_VERSION=latest
     - php: 7.0
-      env: WP_VERSION=4.6.4 WP_MULTISITE=1
```

Comment:
I think on PHP7 to ensure our code will continue to work, we should test the following:
- `WP_VERSION=4.6.5 WP_MULTISITE=1`
- `WP_VERSION=latest WP_MULTISITE=1`
- `WP_VERSION=trunk WP_MULTISITE=1`